types/braintree/tslint.json

```diff
@@ -2,6 +2,8 @@
     "extends": "@definitelytyped/dtslint/dt.json",
     "rules": {
         "no-empty-interface": false,
-        "strict-export-declare-modifiers": false
+        "no-unnecessary-class": false,
```

Comment:
Do not add these extra rules.. instead fix the errors.

Reply:
not sure about the whitespace error though, I've seen a lot of packages add this rule, and it seems lint / format is removing the whitespace and then tslint is angry for not having them - maybe I'm missing something

Reply:
Would like another review please  🙏🏻